src/ucs/sys/ucs_netlink.c

```diff
@@ -0,0 +1,119 @@
+#ifdef HAVE_CONFIG_H
+#  include "config.h"
+#endif
+
+#include "ucs_netlink.h"
+
+#include <string.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <unistd.h>
+#include <errno.h>
+#include <arpa/inet.h>
+#include <net/if.h>
+#include <linux/netlink.h>
+#include <ucs/type/status.h>
+#include <ucs/debug/log.h>
```

Comment:
1) order alphabeticly
2) check if some includes can be removed
3) same in other files

Reply:
unresolved